runtime/sync_win32.h

```diff
@@ -0,0 +1,133 @@
+/**************************************************************************/
+/*                                                                        */
+/*                                 OCaml                                  */
+/*                                                                        */
+/*                         Antonin Decimo, Tarides                        */
```

Comment:
```suggestion
/*                         Antonin Décimo, Tarides                        */
```

Reply:
I'd like to, but I think we're only allowed ASCII. IIRC there's a Hygiene check that prevents this. TBH I'm not even sure my name officially has an accent.

Reply:
Ah, right.

Reply:
check-typo won’t like that, will it?